packages/core-p2p/lib/peer.js

```diff
@@ -76,16 +76,7 @@ module.exports = class Peer {
     try {
```

Comment:
This try/catch won't do anything anymore because __broadcastTransactions already catches any exceptions.